docker-compose.yml

```diff
@@ -52,6 +54,7 @@ services:
       C_FORCE_ROOT: "true"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+      - ./settings.py:/code/settings.py
```

Comment:
If this file doesn't exist will docker overwrite the existing file with nothing?

Reply:
It will actually create a directory called `settings.py` in the filesystem local to the docker-compose file and mount that directory